packages/discovery-react-components/src/utils/document/transformEnrichment.ts

```diff
@@ -38,46 +52,37 @@ function setMetadata(enrichedHtml: Contract): void {
       });
     }
   });
+
   enrichedHtml.metadata = updatedMetadata;
-}
 
-function getOntology(enrichmentName: string): Ontology | undefined {
-  let ontology;
-  if (enrichmentName === ENRICHMENTS.INVOICE) {
-    ontology = invoiceOntology;
-  } else if (enrichmentName === ENRICHMENTS.PURCHASE_ORDER) {
-    ontology = purchaseOrderOntology;
-  }
-  return ontology;
+  return enrichedHtml;
 }
 
-function setAttributesAndRelations(doc: any, enrichmentName: string): void {
-  const ontology = getOntology(enrichmentName);
-  if (ontology) {
-    doc.attributes = createAttributeObjects(ontology.attributes, doc);
-    doc.relations = createRelationObjects(ontology, doc);
-  }
+function setAttributesAndRelations(
+  enrichment: Invoice | PurchaseOrder,
+  enrichmentName: string
+): Invoice | PurchaseOrder {
+  const ontology = ontologyMapping[enrichmentName];
+  enrichment.attributes = setAttributes(ontology.attributes, enrichment);
```

Comment:
I think this would be cleaner if `setAttributes` took `ontology` directly, then within that function found the `ontology.attributes` prop.

Reply:
Agreed, this would make it aligned with the setRelations function.